db/config.cc

```diff
@@ -567,7 +588,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "Time interval in milliseconds to reset all node scores, which allows a bad node to recover.")
     , dynamic_snitch_update_interval_in_ms(this, "dynamic_snitch_update_interval_in_ms", value_status::Unused, 100,
         "The time interval for how often the snitch calculates node scores. Because score calculation is CPU intensive, be careful when reducing this interval.")
-    , hinted_handoff_enabled(this, "hinted_handoff_enabled", value_status::Used, "true",
+    , hinted_handoff_enabled(this, "hinted_handoff_enabled", value_status::Used, db::hints::host_filter(db::hints::host_filter::enabled_for_all_tag()),
```

Comment:
Use `db::config::hinted_handoff_enabled_type` instead of `db::hints::host_filter`

Reply:
Done